src/main/java/org/itstep/j2_16/service/EmployeeService.java

```diff
@@ -12,4 +12,6 @@
     Employee save(Employee employee);
 
     Employee update(long id, Employee employee);
+
+    public float calculateYearlySalary(Employee employee);
```

Comment:
Plz, google for the correct translation:

https://translate.google.com/#ru/en/%D0%B3%D0%BE%D0%B4%D0%BE%D0%B2%D0%BE%D0%B9%20%D0%BE%D0%BA%D0%BB%D0%B0%D0%B4